packages/web-components/src/components/cbp-toggle/cbp-toggle.scss

```diff
@@ -1,17 +1,230 @@
 /**
- * @Prop   --cbp-toggle-XXX
+//TODO: rename cssvars using the --cbp-toggle-color-  format (I.E. --cbp-toggle-color-bg)
+ * @Prop   --cbp-toggle-color-bg: var(--cbp-color-interactive-secondary-light);
+ * @Prop   --cbp-toggle-color-bg-dark: var(--cbp-color-interactive-secondary-dark);
+ * @Prop   --cbp-toggle-color-circle: var(--cbp-color-white);
+ * @Prop   --cbp-toggle-color-circle-dark: var(--cbp-color-white);
+ * @Prop   --cbp-toggle-color-circle-border: var(--cbp-toggle-color-circle);
+ * @Prop   --cbp-toggle-color-circle-border-dark: var(--cbp-toggle-color-circle-dark);
+ * @Prop   --cbp-toggle-outline-width: 2px;
+ * @Prop   --cbp-toggle-outline-style: solid;
+ * @Prop   --cbp-toggle-outline-color: transparent;
+ * @Prop   --cbp-toggle-outline: var(--cbp-toggle-outline-width) var(--cbp-toggle-outline-style) var(--cbp-toggle-outline-color);
+ * @Prop   --cbp-toggle-outline-color-dark: 2px solid transparent;
+ * @Prop   --cbp-toggle-outline-dark: var(--cbp-toggle-outline-width) var(--cbp-toggle-outline-style) var(--cbp-toggle-outline-color-dark);
+ * @Prop   --cbp-toggle-color-bg-hover: var(--cbp-color-interactive-secondary-dark);
+ * @Prop   --cbp-toggle-color-bg-hover-dark: var(--cbp-color-interactive-secondary-base);
+ * @Prop   --cbp-toggle-circle-selected-color-hover: var(--cbp-color-interactive-selected-dark);
+ * @Prop   --cbp-toggle-circle-selected-color-hover-dark: var(--cbp-color-interactive-selected-light);
+ * @Prop   --cbp-toggle-circle-selected-border-color-hover: var(--cbp-color-white);
+ * @Prop   --cbp-toggle-circle-selected-border-color-hover-dark: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-toggle-color-circle-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-toggle-color-circle-hover-dark: var(--cbp-color-interactive-secondary-base);
+ * @Prop   --cbp-toggle-color-circle-border-hover: var(--cbp-color-white);
+ * @Prop   --cbp-toggle-color-circle-border-hover-dark: var(--cbp-color-white);
+ * @Prop   --cbp-toggle-color-bg-focus: var(--cbp-color-interactive-focus-dark);
+ * @Prop   --cbp-toggle-color-bg-focus-dark: var(--cbp-color-interactive-focus-light);
+ * @Prop   --cbp-toggle-outline-focus: 2px solid var(--cbp-color-white);
+ * @Prop   --cbp-toggle-outline-focus-dark: 2px solid var(--cbp-color-black);
+ * @Prop   --cbp-toggle-color-bg-disabled: var(--cbp-color-interactive-disabled-dark);
+ * @Prop   --cbp-toggle-color-bg-disabled-dark: var(--cbp-color-interactive-disabled-light);
+ * @Prop   --cbp-toggle-color-circle-disabled: var(--cbp-color-interactive-disabled-light);
+ * @Prop   --cbp-toggle-color-circle-disabled-dark: var(--cbp-color-interactive-disabled-dark);
+ * @Prop   --cbp-toggle-color-circle-border-disabled: var(--cbp-color-interactive-disabled-light);
+ * @Prop   --cbp-toggle-color-circle-border-disabled-dark: var(--cbp-color-interactive-disabled-dark);
+ * @Prop   --cbp-slider-width: 3.25rem;
```

Comment:
slider is a bad name for this. It needs to be namespaced with cbp-toggle. But since that's the full component, it probably makes sense to clarify exactly what you are applying the width to (not the full host element). I would call it "control", e.g.,  `--cbp-toggle-control`. All the others below need to keep the `--cbp-toggle` before whatever part they are describing.

Reply:
updated instances of --cbp-slider to --cbp-toggle-control